.dev/docker/windows/Dockerfile

```diff
@@ -34,8 +34,10 @@ RUN wget $Env:CHANNEL_BASE_URL/vs_buildtools.exe -OutFile 'C:\TEMP\vs_buildtools
     -Wait -PassThru;                                                `
     del c:\temp\vs_buildtools.exe;                                  
 
-RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'));        `
-    choco install cmake git --installargs 'ADD_CMAKE_TO_PATH=System' -y --no-progress
+# VCPKG requires update if Cmake version is > 3.20.5 see: https://github.com/microsoft/vcpkg-tool/pull/107
```

Comment:
That PR is included in https://github.com/microsoft/vcpkg-tool/releases/tag/2021-07-16

Reply:
Yes, but the commit of https://github.com/microsoft/vcpkg that we are using, doesn't download that new VCPKG executable. We can update VCPKG, but then we'll also update some of the libraries (ie. boost and some other minor sublibaries).
So to have a larger span on tested libraries we "paused" the x86 on a specific commit.